### PR TITLE
Fix AuthN filters.

### PR DESCRIPTION
**Pull request by pangea-andrest**
- They haven't been serialized properly due to filter type was a Union[], so .dict() method wasn't called recursively

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Rewrote `README.md`.
-- Audit.search() `order_by` param also takes `str` now
+- Audit.search() `order_by` param also takes `str` now.
+- Fixed AuthN list method's filter. Filters have not been serialized properly.
```
Comment (kenany) [inline CHANGELOG.md:20]: Maybe?:
```suggestion
- Fixed AuthN list method's filter not being serialized properly.
```

And do we want this under a `### Fixed` heading?

File: packages/pangea-sdk/tests/integration/test_authn.py [hunk 2]
```diff
@@ -255,7 +265,7 @@ def test_authn_c2_login_n_client_session_invalidate(self):
             token = response_login.result.active_token.token
 
             # list sessions
-            response = self.authn.client.session.list(token=token)
+            response = self.authn.client.session.list(token=token, filter={})
```
Comment (kenany) [inline packages/pangea-sdk/tests/integration/test_authn.py:268]: Are these empty filter dicts necessary now?

File: packages/pangea-sdk/tests/integration/test_authn.py [hunk 3]
```diff
@@ -160,7 +160,7 @@ def test_authn_a5_user_update(self):
         self.assertEqual(False, response.result.disabled)
 
     def test_authn_b1_user_invite(self):
-        # This could (should) fail if test_authn_user_create_with_password failed
+        # This could (should) fail if test_authn_user_createf_with_password failed
```
Comment (kenany) [inline packages/pangea-sdk/tests/integration/test_authn.py:163]: I don't see anything with either of these names.

File: packages/pangea-sdk/tests/integration/test_authn.py (hunk 2 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/tests/integration/test_authn.py:268]: It's optional, it was just for testing that also accept a dict, even if it's empty

File: packages/pangea-sdk/tests/integration/test_authn.py (hunk 3 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/tests/integration/test_authn.py]: fixed

**Review by kenany — APPROVED** [review]

**Pull request merged**